tools/deb/envoy_bootstrap_v2.json

```diff
@@ -40,7 +40,30 @@
         "tag_name": "listener_address",
         "regex": "^listener\\.(((?:[_.[:digit:]]*|[_\\[\\]aAbBcCdDeEfF[:digit:]]*))\\.)"
       }
-    ]
+    ],
+    "stats_matcher": {
+      "inclusion_list": {
+        "patterns": [{
+            "prefix": "cluster_manager"
+          },
+          {
+            "prefix": "listener_manager"
+          },
+          {
+            "prefix": "http_mixer_filter"
+          },
+          {
+            "prefix": "tcp_mixer_filter"
+          },
+          {
+            "prefix": "server"
+          },
+          {
+            "prefix": "cluster.xds-grpc"
```

Comment:
why not stick with just this? others seem like arbitrary choice.

Reply:
The prefixes were selected for a few reasons:
- `cluster_manager` and `listener_manager` metrics are used by the `pilot-agent` for health checking (so, without them, the container is never ready)
- the `mixer_filter` metrics are used in e2e tests and some dashboards
- i *thought* the `server` metrics were being used to track version skew, etc.